v2/googlecloud-to-elasticsearch/src/main/java/com/google/cloud/teleport/v2/elasticsearch/templates/BigQueryToElasticsearch.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (C) 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.elasticsearch.templates;
+
+import com.google.cloud.teleport.v2.elasticsearch.options.BigQueryToElasticsearchOptions;
+import com.google.cloud.teleport.v2.elasticsearch.options.ElasticsearchWriteOptions;
+import com.google.cloud.teleport.v2.elasticsearch.transforms.WriteToElasticsearch;
+import com.google.cloud.teleport.v2.transforms.BigQueryConverters.ReadBigQuery;
+import com.google.cloud.teleport.v2.transforms.BigQueryConverters.TableRowToJsonFn;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.PipelineResult;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.transforms.ParDo;
+
+/**
+ * The {@link BigQueryToElasticsearch} pipeline exports data from a BigQuery table to Elasticsearch.
+ *
+ * <p><b>Pipeline Requirements</b>
```

Comment:
Below description can be removed and replace with Link to Readme.md file since it was observed in the past that the description in the current file and Readme.md diverges along with additional changes.

Same comment applies to rest of the templates as well

Reply:
Since README.md is not a source file it's problematic to reference it correctly using Javadoc. May I just put this:  "Please refer to <b>README.md</b> for further information." ?

Reply:
Yes that is the intention. Make README.md as link (e.g:  <a href="/path/to/readme.md">README.md</a> pointing to respective file

Reply:
Done.